sdk/appconfiguration/Azure.Data.AppConfiguration/src/Models/SnapshotSettingFilter.cs

```diff
@@ -7,12 +7,13 @@
 namespace Azure.Data.AppConfiguration
 {
     /// <summary> Enables filtering of key-values. </summary>
-    public partial class ConfigurationSettingFilter
+    [CodeGenModel("ConfigurationSettingFilter")]
```

Comment:
We are not generating models, so why do we need `[CodeGenModel("ConfigurationSettingFilter")]`?